build.sc

```diff
@@ -357,6 +367,11 @@ trait JvmIntegrationCore extends CliIntegrationCore {
   def cliKind = "jvm"
 }
 
+trait DockerIntegrationCore extends CliIntegrationCore {
+  def cliKind = "docker"
+  def testLauncher = `cli-core`.launcher()
```

Comment:
Do we need to pass a test launcher to the docker tests?

Reply:
Probably not, so how should I implement method testLauncher in DockerIntegrationCore?

Reply:
Just don't use `CliIntegrationBase` in `DockerIntegrationCore`.